.svglintrc.js

```diff
@@ -163,6 +163,44 @@ module.exports = {
               }
             }
           },
+          function(reporter, $, ast) {
+            reporter.name = "icon-path";
+
+            const iconPath = $.find("path").attr("d");
+            if (!updateIgnoreFile && isIgnored(reporter.name, iconPath)) {
+              return;
+            }
+
+            const { segments } = parsePath(iconPath);
+
+            const lowerCommand = ['m', 'l', 'h', 'v'];
+            const upperCommand = ['M', 'L', 'H', 'V'];
+            const commands = [...lowerCommand, ...upperCommand];
+            const getInvalidSegments = ([command, coord1, coord2, ...rest], index) => {
+              if (commands.includes(command)) {
+                if (lowerCommand.includes(command) && coord1 === 0 && coord2 === 0) {
```

Comment:
> Something I haven't realized before, is that the H and V commands ony have a number, so something like this

@mondeja I will need to rethink this part of the code. There is more than that.

- For `h` and `v`, `coord2` will always be undefined so `coord2 === 0` is never true in this case.
- For `h` and `v`, when we compare with a previous segments, we need to compare `coord1` with `coord2`
- As you said, I also need to adapt the reported error according to this